test/sample-test.ts

```diff
@@ -2,12 +2,47 @@ import { expect } from "chai";
 import { starknet } from "hardhat";
 import { StarknetContract, StarknetContractFactory } from "hardhat/types/runtime";
 
+const starkwareCrypto = require('../lib/crypto/starkware/crypto/signature/signature.js');
```

Comment:
Why is this imported like this? Why `require` instead of `import`, and why specifying a relative path instead of package name? Is this officialy recommended? Or could you perhaps add a dependency (either a dev dependency or an ordinary one) to package.json?

Reply:
Turns out it's the best approach currently, until a separate package for this is released. Marking as resolved.